Assets/Systems/StatSystem/Stats/Extentions/Vital/StatVital.cs

```diff
@@ -16,46 +16,57 @@ namespace Systems.StatSystem
     */
     public class StatVital : StatAttribute, IStatCurrentValueChanged
     {
-        private int _statCurrentValue; //the stats current value
+        private int _value; //the stats current value
 
         public event EventHandler OnCurrentValueChanged; //event for when the stat value changes  
 
         #region Constructors 
         public StatVital()
         {
-            _statCurrentValue = 0;
+            _value = 0;
         }
         #endregion
 
         #region Properties - Getters/Setters
-        public int CurrentValue
+        public int Max
+        {
+            get { return base.Value; }
+        }
+
+        new public int Value
```

Comment:
Okay I understood that and I guess it doesn't really hurt after looking over the test/example code. Maybe just leave `_value` as `_current` or `_currentValue` so there's differentiation between the two.